src/Stratis.Features.FederatedPeg/Conversion/ConversionRequest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.FederatedPeg
+{
+    public enum ConversionRequestType
+    {
+        Mint,
+        Burn
+    }
+
+    public enum ConversionRequestStatus
+    {
+        Unprocessed,
+        Submitted,
+        Processed
+    }
+
+    public class ConversionRequest : IBitcoinSerializable
```

Comment:
It would be good to at least add a high level summary on what these new classes do and/or are used for?